builtin/providers/bitbucket/client.go

```diff
@@ -0,0 +1,65 @@
+package bitbucket
```

Comment:
At the time there didnt seem to be any good libraries and bitbucket api seemed easier to reference via the struct then going through it all. I also had to implement methods that werent available yet since I work with the bitbucket team.